service/src/main/resources/api/service_openapi.yaml

```diff
@@ -974,6 +1004,30 @@ components:
         items:
           type: string
 
+    ActivityLogFilterUserEmail:
+      name: userEmail
+      in: query
+      description: Text search on the activity log entry user email
```

Comment:
We might want this to be more exact that a text search, or at least to have such an option, otherwise it would be a pain for cases like usernames that are substrings of other usernames. I'm not sure what the best way to represent API-wise but I'm thinking of a version that matches specifically from the beginning of the username, so `def@` doesn't match `abcdef@verily.com`. A regex mode or exact match mode could be other possibilities.

Reply:
Good idea. I'll add an exact match flag.

Reply:
done.